client/layout/activity-panel/activity-outbound-link/README.md

```diff
@@ -0,0 +1,26 @@
+ActivityPanelOutboundLink
+============
+
+`ActivityPanelOutboundLink` creates a styled link for use in the Activity Panel.
```

Comment:
You're using both `ActivityPanelOutboundLink` & `ActivityOutboundLink` here, but I think it should be just `ActivityOutboundLink`.